cmds/docker

```diff
@@ -35,12 +56,21 @@ docker_enter() {
         return 1
     fi
 
+    echo docker run --rm -it --security-opt seccomp=unconfined \
```

Comment:
is this debugging left behind or are you wanting the docker command echo'ed out everytime?

Reply:
debugging left behind, my bad.